st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -81,6 +81,7 @@ def _save_workbook(self, name, def_yaml):
             wb = self._client.workbooks.create(def_yaml)
 
         # Update the workbook definition.
+        # pylint: disable=no-member
```

Comment:
I need to look into the Mistral client code for that one.

I imagine it does some "dynamic code magic". It might be possible to fix that upstream and make it possible for pylint to detect which attributes are available on `Workbook` and `Execution` class.